wiki/Help_Center/zh.md

```diff
@@ -844,7 +844,7 @@ osu! 安装程序自带有 .NET 框架，为了以防万一，您可以[在这
 
 虽然听起来很复杂，但是修复这个问题很简单。
 
-首先，[下载这个压缩包](https://dl.dropboxusercontent.com/u/16332218/goodies.zip)，将它保存在电脑上。
+首先，[下载这个压缩包](https://dl.dropboxusercontent.com/u/16332218/goodies.zip) （链接失效），将它保存在电脑上。
```

Comment:
Don't know how. I'm not that familiar with GitHub, and I'm afraid editing things having the PR applied might mess things up...